### PR TITLE
Appdev 8733 functional changes for new call number column

### PR DESCRIPTION
**Pull request by sharonluong**
This PR makes a handful of functional changes relating to the archival identifier column in `new_call_number_sequences`.

It also deletes a few things from other tests that were extraneous.

### CODE REVIEW

File: app/Models/Collection.php [hunk 1]
```diff
@@ -24,6 +25,18 @@ public function collectionType()
     return $this->belongsTo(CollectionType::class);
   }
 
+  public static function findArchivalIdentifier($collectionId)
```
Comment (cazzerson) [inline app/Models/Collection.php]: Maybe cut this and just check the value of the archival_identifier in an instantiated collection? If null is never allowed, define this as a validator and/or 'NOT NULL' in the migration.

Comment (sharonluong) [reply app/Models/Collection.php]: Got it. I already have it as validation on the request so it should be good to go.

File: app/Models/Collection.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\SoftDeletes;
+use DB;
```
Comment (cazzerson) [inline app/Models/Collection.php]: Is this still needed?

Comment (sharonluong) [reply app/Models/Collection.php]: ah! no, will remove.

**Pull request merged**